lib/yam/agama/agama_base.pm

```diff
@@ -20,4 +20,8 @@ sub post_fail_hook {
     save_and_upload_log('journalctl -u agama-auto', "/tmp/agama-auto-log.txt");
 }
 
+sub post_run_hook {
```

Comment:
We no longer need this as long as we don't use the "LIVETEST" variable, the default password will be used.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17136 set password in pre_run_hook, so you need reset password in post hook now.

Reply:
So it was decided to use "LIVETEST" in the end ? Why ?